community/teams/foundation-board.tt

```diff
@@ -38,6 +38,12 @@
     <li>
       Handle sponsoring and donations
     </li>
+    <li>
+      Encourage ontributions by individuals and organisatons alike
```

Comment:
```suggestion
      Encourage contributions by individuals and organisations alike
```

Reply:
Thanks! I love typo fixes because it shows someone actually reads that stuff.